apps/aechannel/src/aesc_fsm.erl

```diff
@@ -207,6 +208,11 @@ shutdown(Fsm) ->
     lager:debug("shutdown(~p)", [Fsm]),
     gen_statem:call(Fsm, shutdown).
 
+client_died(Fsm) ->
+    %TODO: possibility for reconnect
+    lager:debug("client died(~p)", [Fsm]),
+    ok = gen_statem:stop(Fsm).
```

Comment:
This is still the nice way of stopping it, a exit(Pid, kill) would be rougher. But since you don't care about whether or not you can recover the state nicely, this should work fine as a test.